### PR TITLE
Import export fixes

### PR DESCRIPTION
**Pull request by znarf**

### CODE REVIEW

File: server/lib/import-export/export.ts [hunk 1]
```diff
@@ -184,5 +184,6 @@ export const traverse = async (
 export const getMigrationsHash = async () => {
   const [data] = await sequelize.query('SELECT name FROM "SequelizeMeta" ORDER BY name');
   const migrationNames = data.map(d => d.name);
+  migrationNames.sort();
```
Comment (kewitz) [inline server/lib/import-export/export.ts:187]: 👍🏼

File: server/lib/import-export/import.ts [hunk 2]
```diff
@@ -98,7 +98,7 @@ const modelsDeduplicationSchema: Record<ModelNames, { unique?: string[] }> = {
   MigrationLog: {},
   Notification: {},
   OAuthAuthorizationCode: {},
-  Order: { unique: ['totalAmount', 'currency', 'createdAt', 'quantity', 'status', 'interval'] },
+  Order: { unique: ['totalAmount', 'currency', 'createdAt', 'quantity', 'status', 'interval', 'description'] },
```
Comment (znarf) [inline server/lib/import-export/import.ts:101]: I noticed undesired conflicts without description in the key.

File: server/lib/import-export/export.ts (hunk 1 above)
Comment (znarf) [inline server/lib/import-export/export.ts:187]: The sorting was not consistent without this.

File: server/lib/import-export/import.ts [hunk 3]
```diff
@@ -157,21 +157,24 @@ export const remapPKs = async (dataFile: string): Promise<PKMap> => {
...
-          return;
-        }
-        // Else, we need remap to their existing PK
-        else {
-          debug(
-            `Record ${record.model}#${record[primaryKey]} already exists with different id ${existingRecord[primaryKey]}`,
-          );
-          pkMap[record.model][record[primaryKey]] = existingRecord[primaryKey];
-          return;
+      const hasNonNullValue = Object.values(where).some(value => !!value);
```
Comment (znarf) [inline server/lib/import-export/import.ts:160]: I noticed a case where PaymentMethod had no uuid exported, ending up with an empty where clause, wrongly matching any record in the table.

**Review by kewitz — APPROVED** [review]

**Review by Betree — APPROVED** [review]

**Pull request merged**